src/helm/benchmark/scenarios/melt_scenarios.py

```diff
@@ -499,3 +500,229 @@ def get_instances(self, output_path: str) -> List[Instance]:
                 instances.append(instance)
 
         return instances
+
+
+class MELTTextClassificationScenario(Scenario):
+    name = "melt_text_classification"
+    description = "Text Classification scenario."
+    tags = ["text_classification"]
+
+    def __init__(
+        self,
+        dataset_name: str,
+        revision: str,
+        subset: Optional[str] = "",
+        text_key: str = "text",
+        label_key: str = "label",
+        splits: Optional[Dict[str, str]] = None,
+    ):
+        """
+        Initializes the question answering scenario.
+
+        Args:
+            dataset_name: The name of the dataset.
+            revision: The revision of the dataset to use.
+            subset: The subset of the dataset to use. Defaults to "".
+            text_key: The key to use for the text in the dataset. Defaults to "text".
+            label_key: The key to use for the label in the dataset. Defaults to "label".
+            splits: The splits to use for the dataset. Defaults to None.
+        """
+        super().__init__()
+        self.dataset_name = dataset_name
+        self.subset = subset
+        self.revision = revision
+        self.text_key = text_key
+        self.label_key = label_key
```

Comment:
nit: `self.text_key` and `self.label_key` are only used by the subclasses. You can just make these class constants in the subclasses, and remove them from the arguments here.